### PR TITLE
Display GTFS errors

### LINKED ISSUE #283: Report errors when uploading GTFS
Upon uploading a ZIP of GTFS and OSM, the progress spinner just keeps spinning indefinitely. I see errors in the server-side logs including some uncaught exceptions, but the new bundle is visible in the UI, and I can't tell if the errors have somehow corrupted or affected the upload.

If an exception is caught on the server side, we should set the upload status to "error" with a message containing the stack trace.

### LINKED ISSUE #1011: Opportunity dataset still selected after deletion
When deleting an opportunity dataset, the dataset and variable name remain selected in the dropdown. It is removed from the map though and doesn't reappear if you click the X next to it. 
The selection should be cleared automatically.

### PR DESCRIPTION
**Pull request by trevorgerhardt**
Display stored errors generated while processing GTFS feeds.

- [x] Add `.txt` to the filenames
- [x] Reformat the error type names so that `MissingTableError` would be displayed as `Missing Table`
- [x] Hide Bundles with `status === 'error'` or that have any `Priority.HIGH` errors.

Additionally, adds an experimental `Combobox` component that could eventually replace the `Select` component. It's not currently used in the application, but there's a test page added that uses it.

fixes #1011, fixes #283

## How to test

1. Create a bundle with a GTFS feed that will contain errors.
2. View the errors after the bundle creation is complete.

### CODE REVIEW

**Comment by trevorgerhardt** [comment]
#### 📝Changed routes:


| [**/debug/select**](https://analysis-esjgrvb4b-conveyal.vercel.app/debug/select) | [**…Id\]/bundles/\[bundleId\]**](https://analysis-esjgrvb4b-conveyal.vercel.app/regions/[regionId]/bundles/[bundleId]) |
|:-:|:-:|
|<a href="https://analysis-esjgrvb4b-conveyal.vercel.app/debug/select">Screenshot of /debug/select</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Yx5OHBAyx1QBx3NIMX8L49VIz6yK2331ALf6td6S6fQ60kO1HJ8RcCj8TJiMdm1AyH9h7LRPgIEEq9eUgj__kA.png">(view full size)</a> |<a href="https://analysis-esjgrvb4b-conveyal.vercel.app/regions/[regionId]/bundles/[bundleId]">Screenshot of /regions/[regionId]/bundles/[bundleId]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/D_Kl7dbhxHVbzGgo1TH2kyprgQQTHg-3_sKk9sMpU2cE1gXmmZJaTJUFis66lJopn0LaURsAv7qM5n6PJ0WHVA.png">(view full size)</a> |



| [**…ons/\[regionId\]/bundles**](https://analysis-esjgrvb4b-conveyal.vercel.app/regions/[regionId]/bundles) |
|:-:|
|<a href="https://analysis-esjgrvb4b-conveyal.vercel.app/regions/[regionId]/bundles">Screenshot of /regions/[regionId]/bundles</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/1H21qHqmYlHhfiF8E3pZlPQzA7dF3O8voa9tXnei5cb0HccXZSXoq-CDZynQFY4tSeotgb6I4NBNRsR-PB0rsw.png">(view full size)</a> |





2 deleted routes:
- **/regions/[regionId]/bundles/[bundleId]**
- **/regions/[regionId]/bundles**

Commit 1bf383966a7adc166d12adb453772fa680f30321 (https://analysis-esjgrvb4b-conveyal.vercel.app).

File: lib/components/create-project.tsx [hunk 1]
```diff
@@ -25,6 +25,15 @@ const hasText = (s) => s && s.length > 0
 const getName = fpGet('name')
 const getId = fpGet('_id')
 
+function filterInvalidBundles(bundles: CL.Bundle[]): CL.Bundle[] {
+  return bundles.filter((b) => {
+    return (
+      b.status === 'DONE' &&
+      b.feeds.every((f) => f.errors.every((e) => e.priority !== 'HIGH'))
```
Comment (ansoncfit) [inline lib/components/create-project.tsx:32]: After uploading the feed with invalid references (shared previously), the create project page shows this error: `Cannot read property 'every' of undefined` (https://analysis-git-display-gtfs-errors-conveyal.vercel.app/regions/5a2e9df732b98e221fef89eb/create-project)

File: lib/components/edit-bundle.tsx [hunk 2]
```diff
@@ -1,46 +1,161 @@
...
+              </HStack>
+            ))}
+          </Stack>
+        </Collapse>
+      </Box>
 
-function FeedNameInput({feed, index, onChange, ...p}) {
+      {error.count > 10 && (
+        <Text textAlign='center' fontStyle='italic' mt={2} width='100%'>
+          Note: only the first ten errors of each type are saved.
```
Comment (ansoncfit) [inline lib/components/edit-bundle.tsx:111]:
```suggestion
          plus {error.count - 10} more errors of this type.
```

Also suggest moving this into the collapse above -- shouldn't need to be shown when boxes are collapsed.

File: lib/components/edit-bundle.tsx [hunk 3]
```diff
@@ -1,46 +1,161 @@
...
+  const name = formatType(error.type)
+  return (
+    <Alert
+      alignItems='flex-start'
+      flexDir='column'
+      status={getSummaryStatus(error)}
+    >
+      <Flex mb={isOpen ? 2 : 0} width='100%'>
+        <AlertIcon />
+        <AlertTitle flex='2' fontSize='xl'>
```
Comment (ansoncfit) [inline lib/components/edit-bundle.tsx:63]: Add `onClick={onToggle}` to the title too, so users don't have to click precisely on the chevron?

**Review by ansoncfit — CHANGES_REQUESTED** [review]
I like the error summary display. I flagged one issue and a few small suggestions below.

File: lib/components/create-project.tsx (hunk 1 above)
Comment (trevorgerhardt) [reply lib/components/create-project.tsx]: Should be fixed 👍

**Review by ansoncfit — APPROVED** [review]
Thanks for incorporating the suggestions.

**Pull request merged**